File: cash.py
import re
from cs50 import get_float

while True:
    coins = get_float("Change: ")
    if coins > 0:
        break
print(coins)
coins = coins * 100    

c = 0

while coins >= 0:
    if coins >= 25:
        c = c + 1
        coins = coins - 25
    elif coins >= 10:
        c = c + 1
        coins = coins - 10
    elif coins >= 5:
        c = c + 1
        coins = coins - 5
    else:
        c = c + 1
        coins = coins - 1
    
print(c)

File: pset1/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    float cents;
    do
    {
        cents = get_float("how much change? ");//prompts how much change u have//
    }
    while (cents < 0);
    
    int n = cents * 100;//converts change to 100x//
    
    n = round(n);

    
    int c = 0;
    while (n >= 0) //series of steps to go trhough algortihm until it hits zero//
    {
        if (n >= 25)
        {
            n = n - 25;
            c++;
        }
        else if (n >= 10)
        {
            n = n - 10;
            c++;
        }
        else if (n >= 5)
        {
            n = n - 5;
            c++;
        }
        else
        {
            n = n - 1;
            c++;
            
        }
    
    }
    printf("%i\n", c - 1); //prints teh number of coins needed//

    
    
}